executors/node/localedisplaynames.js

```diff
@@ -35,6 +35,12 @@ module.exports = {
         "error_type": "unsupported",
         "error_retry": false  // Do not repeat
       };
+      if (error instanceof RangeError) {
+        // The locale can't be handled for some reason!
+        outputLine["error_type"] = 'unsupported';
+        outputLine["unsupported"] = error.toString();
+        outputLine["error_detail"] = 'unsupported locale';
+      }
```

Comment:
`error instanceof RangeError` seems a bit broad. I suggest a narrower case for returning unsupported.

Reply:
Hmm, there are two types of failures right now. Here's what I see being caught as exceptions:

1. Unexpected options in the locale spec, e.g., "label":"0009","language_label":"**en-u-nu-deva-t-de**","locale_label":"en"

  --> "RangeError: invalid_argument"

2. "root" locale --> "RangeError: invalid_argument" in the constructor.

Should either of these be "unsupported"?

Reply:
I think classifying these as "unsupported" is correct.  This will resolve a large number of "errors" --> "unsupported", i.e.,

Report created: 2024-10-07 10:36

1,990 attempted. Pass: 1,460, Fail: 214, **Errors: 316,** Unsupported: 0

Reply:
PTAL!